### DISCUSSION TITLE
It would be better if we promote it  as  a 'windowline' plugin rather than 'statusline' plugin

### DISCUSSION THREAD

**Discussion by milanglacier** [b0o/incline.nvim#10] [General]
Promoting this plugin as a `statusline` might lead to some misconfusion.
Most people are accustomed to `lualine` and might unwilling to make switch  (if they think this is a statusline plugin) since most of the neovim colorscheme has supports for lualine statusline but not many have support to other statusline plugin.
However, since what this plugin doing is actually "window-line", it is just adding a float window for each window to show some lite information, and it is completely compatible with lualine, 
it would be better if we promote this plugin as a "window line" that is as a complement of the current statusline plugin in the `laststatus = 3` scenerio where there is a shortage which is we can not easily differentitiate the filename of different window

**Comment by b0o** [comment]
Hi @milanglacier! You make a great point, and this is something I've been concerned about too. Apart from avoiding confusion, another positive aspect of picking a new category name is that people will want to click through to find out what a "windowline" is. 

I am currently using the term "winline" [in the code](https://github.com/b0o/incline.nvim/blob/main/lua/incline/winline.lua). Do you think "windowline" or "winline" is preferable? "windowline" is more clear, but "winline" is shorter and sort of fun to say. Alternatively, I am open to other suggestions!

**Comment by milanglacier** [comment]
I think `winline` is better.
Anyway, win is short for window, this is consensus.
